client/src/views/files/FileOperationMenu.vue

```diff
@@ -236,6 +253,17 @@ async function cancelOperation(importId: string): Promise<void> {
   await fileOperationManager.cancelOperation(importId);
 }
 
+async function cancelQueuedOperations(): Promise<void> {
+  errorHappenedInUpload.value = true;
+
+  for (const operation of queuedItems.value) {
```

Comment:
Only call `cancelOperation`. The fileOperationManager should then raise an event FileOperationState.Cancelled to warn you that the operation has been cancelled, at which time you can remove it from one of your queues.